SR2 Community Patch/scripts/menu/options_menu.as

```diff
@@ -224,6 +224,16 @@ class GameOptions : OptionsBox {
 			locale::OPT_MENU_BG_SCREENSHOT, "bMenuBGScreenshot"
 		));
 
+		y += 38;
+		GuiEngineNumber quickbarMinLabor(
+			panel, recti_area(8, y,  550, 28),
+			locale::OPT_QUICKBAR_MINLABOR,
+			"dQuickbarMinLabor"
+		);
+		quickbarMinLabor.decimals = 0;
```

Comment:
The default should probably be 1 right? To fix the issue with Mechanoid empires?

Reply:
this just sets the amount of decimals to 0. good point, it should probably be a number field after all, since it only takes `int`s anyway...

Reply:
He's got a point, though. The default value in `settings.txt` should perhaps be 1.

Reply:
2, because it is a >= check

Reply:
No it's not.

Reply:
Oh, it *was* a >= check after this commit...

Anyway, yeah, if there's an integer field (I'm a little rusty after not doing any modding all semester :cry:), I'd suggest switching over to that instead of messing with decimal fields.

Reply:
> Oh, it _was_ a >= check after this commit...
> 
> Anyway, yeah, if there's an integer field (I'm a little rusty after not doing any modding all semester 😢), I'd suggest switching over to that instead of messing with decimal fields.

I should have made this a review comment.